docs/Developers/Newcomers_Guide.rst

```diff
@@ -87,7 +87,39 @@ Step 1. Meet the Community!
 
 To get started, the first step is to meet the community. We use gitter to
 communicate, and there the helpful community will guide you.
-Join us at `coala gitter <https://coala.io/chat>`_.
+Gitter is instant messaging service used by developers and users of GitHub.
+Gitter uses chatrooms, where developers can join in and can talk about
+particular topic.
+Coala has 2 types of chatrooms - repository chatrooms and discussion topics.
+Repository chatrooms are related to specific repository and
+Discussion chatrooms are related to general discussion topics like
+conferences, workshops, etc.
+
+  * `coala <https://gitter.im/coala/coala>`_
+    This is the main chatroom and repository chatroom of coala/coala.
+  * `gsoc <https://gitter.im/coala/coala/gsoc>`_
+    This is where you discuss about Google Summer of Code.
+  * `coala-bears <https://gitter.im/coala/coala-bears>`_
+    Repo chatroom for coala/coala-bears.
+  * `workshop <https://gitter.im/coala/coala/workshops>`_
+    Discussions related to workshops go here.
+  * `conferences <https://gitter.im/coala/conferences>`_
+    Everything related to conferences.
+  * `cobot <https://gitter.im/coala/cobot>`_
```

Comment:
I would remove the cobot channel, because at this point in the document cobot was never introduced, so people don't know what you're talking about.

Rather add the offtopic channel as a place where people can talk about anything.